storybook/src/Omar/Atoms/Inputs/Inputs.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Inputs } from "./Inputs";
+
+describe("should change input text", () => {
+  it("show what user write", () => {
+    const inputMock = jest.fn();
+    render(<Inputs onChange={inputMock} />);
```

Comment:
The use of mocking here has not effect. If you removed the mock fn, the expected behavior would work normally. 
Please add another test case using the onChange listener properly.

Reply:
Okay, i've changed it.